cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -86,6 +95,8 @@ def _init_options(self, kwargs):
         if process_bool_arg(self.options.get("pdb")):
             patch_statusreporter()
 
+        self.options.setdefault("sources", {})
```

Comment:
From what I see on lines 60-62 and 115, it looks like `sources` is meant to be a list rather than a dict.

Reply:
good catch